requirements.txt

```
@@ -1,5 +1,5 @@
-docutils==0.20.1
-Sphinx==7.2.6
-sphinx-rtd-theme==2.0.0
+docutils==0.21.2
+Sphinx==8.2.3
+sphinx-rtd-theme==3.0.2
 readthedocs-sphinx-ext==2.2.5
-Jinja2==3.1.5
+Jinja2==3.1.6
```

had to go for 
```suggestion
docutils==0.20.1
Sphinx==7.1.2
sphinx-rtd-theme==3.0.2
readthedocs-sphinx-ext==2.2.5
Jinja2==3.1.6
```
on mac :( , but hey, at least I found a working combination again.